<?xml version="1.0"?>
<launch>
    <include file="$(find ros_odrive)/launch/urdf.launch"/>

    <node name="ros_odrive" pkg="ros_odrive" type="ros_odrive" required="true" output="screen">
        <param name="rate" value="10"/>
        <param name="od_sn" value="0x335831593536"/>
        <!--    <remap from="ros_odrive/odom" to="odom"/>-->
    </node>

    <node pkg="ros_odrive" type="robot_pose_publisher" name="rosgolf_pose_publisher">
        <param name="is_stamped" value="true"/>
    </node>

    <!--  <node name="urm_37_v40" pkg="urm_37_v40" type="node" output="screen"/>-->

    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" >
        <arg name="port" value="9090"/>
    </include>

    <include file="$(find ros_odrive)/launch/webui.launch"/>

</launch>